string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
+function firstChars(length, string) {
+  var newString = string.substring(0, length);
+  var tempArray = newString.split(' ');
+  var updatedString = '';
+  for (var i = 0; i < tempArray.length; i++) {
+    if (tempArray[i] === tempArray[i].toUpperCase()) {
+      var newWord = tempArray[i];
+    } else if (tempArray[i] === 'LearningFuze') {
+      newWord = tempArray[i];
+    } else {
+      var endOfWord = tempArray[i].substring(1, tempArray[i].length);
+      endOfWord = endOfWord.toLowerCase();
+      var firstLetter = tempArray[i][0].toUpperCase();
+      newWord = firstLetter + endOfWord;
+    }
+    updatedString = updatedString + ' ' + newWord;
+  }
```

Comment:
If you slice the string from `0` to `length`, you'll get the first `length` chars.

Reply:
Went from 18 lines of code to 4... thats crazy